process_data_set_irene.R

```diff
@@ -0,0 +1,73 @@
+library(tidyverse)
```

Comment:
This file needs to be in `data-raw/`, not at the root level of the repository.

Reply:
You have to delete this file and then commit the deletion to get it out of here.